CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Fixes
 - Fix prop types of `Tooltip` component @kuzhelov ([#1499](https://github.com/stardust-ui/react/pull/1499))
 - Fix `Popup` styling with `pointer` when it is wrapped by `FocusZones` @layershifter ([#1492](https://github.com/stardust-ui/react/pull/1492))
+- Fix `Attachment` more option button invoke correctly action on space/enter @kolaps33 ([#1452] (https://github.com/stardust-ui/react/pull/1452))
```

Comment:
do we need to add changelog for a unit test?

Reply:
this is related to the previous PR, where I forgot to update changelog:
https://github.com/stardust-ui/react/pull/1452